tests/integration-tests/processors/processors-test.js

```diff
@@ -9,20 +9,77 @@ describe('Processors api', function() {
         await requestSender.init();
     });
 
-    describe('Good requests', function() {
+    describe('Good requests', async function() {
+        describe('GET /v1/processors', async function () {
+            let numberOfProcessorsToInsert = 101;
+            let jsProcessorsArr = [];
+            let processorsInserted = [];
+            before(async function() {
+                for (let i = 0; i < numberOfProcessorsToInsert; i++) {
+                    const processor = generateRawJSProcessor(i.toString());
+                    jsProcessorsArr.push(processor);
+                    const processorRes = await requestSender.createProcessor(processor, validHeaders);
+                    processorsInserted.push(processorRes);
+                }
+            });
```

Comment:
When using Promise.all to insert all 101 processors, Date.now() returns the same value for almost all processors, which will impact the DESC order, since they all have the same value, the order will be random.
I had to insert it in a waterfall order.
If you have a better way to ensure order, I would love to know :) 
In terms of performance, I didn't see any significant difference.

Reply:
Seems good.  Maybe refactor to different `generateProcessors(numberOfProcessors)` function, but not critical for now